graph/graph.go

```diff
@@ -0,0 +1,292 @@
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strconv"
+
+	"go.viam.com/rdk/components/arm"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/robot/client"
+	"go.viam.com/rdk/spatialmath"
+	"go.viam.com/utils/rpc"
+)
+
+type trajInput struct {
+	Timestamp   float64
+	JointInputs []referenceframe.Input
+}
+
+type trajPoses struct {
+	Timestamp float64
+	Pose      spatialmath.Pose
+}
+
+type config struct {
+	Viam struct {
+		Host     string `json:"host"`
+		EntityID string `json:"entity_id"`
+		APIKey   string `json:"api_key"`
+	} `json:"viam"`
+	ArmName               string `json:"arm_name"`
+	TrajectoryCSV         string `json:"trajectory_csv"`
+	WaypointsCSV          string `json:"waypoints_csv"`
+	OutputTrajectoryPoses string `json:"output_trajectory_poses"`
+	OutputWaypointPoses   string `json:"output_waypoint_poses"`
+}
+
+func loadConfig(path string) (config, error) {
+	var cfg config
+	file, err := os.Open(path)
+	if err != nil {
+		return cfg, err
+	}
+	defer file.Close()
+
+	decoder := json.NewDecoder(file)
+	err = decoder.Decode(&cfg)
+	return cfg, err
+}
+
+func main() {
+	logger := logging.NewDebugLogger("client")
+	ctx := context.Background()
+
+	cfg, err := loadConfig("config.json")
+	if err != nil {
+		log.Fatalf("failed to load config: %v", err)
+	}
+
+	machine, err := client.New(
```

Comment:
We don't need to connect to a machine.  This go code seems like its only for converting to euclidean space right?